detection_rules/config.py

```diff
@@ -150,16 +151,38 @@ class RulesConfig:
     exception_dir: Optional[Path] = None
 
 
+def check_path(content: dict, keys: tuple, content_name: str):
+    """Check if a dict key path exists in the supplied content."""
+    try:
+        reduce(lambda d, key: d[key], keys, content)
+    except KeyError:
+        raise ValueError(f"Missing {'.'.join(keys)} in {content_name} contents.")
+
+
+def validate_package_contents(package_contents: dict):
+    """Validate the contents of the package for required fields."""
+
+    paths = [
+        ('name',),
+        ('registry_data', 'conditions'),
```

Comment:
nothing under registry data should be required (if so we need to decouple) - that is only relevant for EPR fleet releasing.

Reply:
https://github.com/elastic/detection-rules/pull/3653#discussion_r1594303263